browser/src/core/LOUtil.js

```diff
@@ -118,6 +118,21 @@ L.LOUtil = {
 
 		map.on('themechanged', setupIcon, this);
 	},
+	setUserImage: function(img, map) {
+		var docType = map.getDocType();
+		// set avatar image if it exist in user extract info
+		var defaultImage = L.LOUtil.getImageURL('user.svg', docType);
+		var myViewId = map._docLayer._viewId;
+		var extraInfo = map._viewInfo[myViewId].userextrainfo;
+		if (extraInfo !== undefined && extraInfo.avatar !== undefined) {
+			// set user avatar
+			img.src = extraInfo.avatar;
+			img.addEventListener('error', () => (img.src = defaultImage));
```

Comment:
we do a `checkIfImageExists` for the default image normally, but this won't happen if there was meant to be a user image and we fallback to default, this is probably an oversight

Reply:
Added `checkIfImageExist` and a safeguard that will prevent `error` event registration if it already has one
CC: @Minion3665